plugin/src/main/kotlin/org/jetbrains/research/pynose/plugin/inspections/unittest/SuboptimalAssertTestSmellUnittestInspection.kt

```diff
@@ -63,6 +63,8 @@ class SuboptimalAssertTestSmellUnittestInspection : AbstractTestSmellInspection(
                     return
                 }
                 if (CHECKERS.any { checker -> checker(callExpression) }) {
+                    // todo: does not stop highlighting assertTrue(self.X) and other assertions with one parameter
+                    //  but does not highlight them if I just type them down (not using quick fix) and I have no idea why
```

Comment:
Probably, you don't manage PSI in the right way. It throws **NPE** each time when I apply quick-fix in such situation. I guess, this is because you replace `PyCallExpression` with `PyStatement`. Could you try replacing not `PyCallExpression` but its parent, `PyExpressionStatement`?

Reply:
thanks! it helped